db/migrate/20220701173725_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :role, :integer
+  end
```

Comment:
- [ ] Kindly, instead of using integer datatype for the `role column`, I'll suggest you use the string datatype.
After generating a this new column (role) make sure that you run `rails:db migrate` so that your new column created will be migrated in your schema.rb file.